<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-KCS3PNGF" name="com.offline.TestDataProvider" tests="1" failures="1" timestamp="11 Oct 2019 03:38:54 GMT" time="0.000" errors="0">
  <testcase name="getEmployeeData1" time="0.000" classname="com.offline.TestDataProvider">
    <failure type="org.testng.TestNGException" message="
Data Provider public java.util.Iterator[] com.offline.TestDataProvider.testDP1() throws java.io.IOException must return either Object[][] or Iterator&lt;Object&gt;[], not class [Ljava.util.Iterator;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public java.util.Iterator[] com.offline.TestDataProvider.testDP1() throws java.io.IOException must return either Object[][] or Iterator<Object>[], not class [Ljava.util.Iterator;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:156)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:431)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1335)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1051)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1140)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- getEmployeeData1 -->
</testsuite> <!-- com.offline.TestDataProvider -->
